integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -128,6 +128,7 @@ const getSauceLabsConfig = (capabilities) => {
     name: capabilities.name,
     tags: [capabilities.name],
     seleniumVersion: '3.141.59',
+    screenResolution: '1280x960',
```

Comment:
Does this work in mobile? Have you verified?

Reply:
I think the better fix is to fix the CSS in the meeting demo. Could you please first try that?

Reply:
This method is not used in mobile. Please let me know if I missed something.

Reply:
Ok confirmed that this method is not used for mobile so I am good. @devalevenkatesh Think we can use this for now and fix CSS later (we probably want to own it) :)

Reply:
Cool, yup agree.